sql/driver/conn.go

```diff
@@ -178,13 +178,18 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.IntVal != nil {
 				t[j] = *datum.IntVal
 			} else if datum.UintVal != nil {
-				t[j] = *datum.UintVal
+				// uint64 not supported by the driver.Value interface.
```

Comment:
Yeah that would be a nicer way to put this code behind an interface. @petermattis any thoughts about that. I can put that in another PR.